src/Components/GenreSelect/GenreSelect.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { buttonStyle, containerStyle, selectedButtonStyle } from "./styles";
+
+export const GenreSelect = ({ allGenres, selectedGenre, onSelect }) => {
+    const handler = (e) => {
+        onSelect(e.target.name);
+    };
+
+    return (
+        <div style={containerStyle}>
+            {allGenres.map((genre) => (
+                <button
+                    key={genre}
+                    onClick={handler}
+                    name={genre}
```

Comment:
Using `name` on `button` like this doesn't seem correct - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#name.

For now let's inline the handler like this:

```suggestion
                    onClick={() => onSelect(genre)}
                    name={genre}
```

Will get back to it in later modules.